### PR TITLE
Slashes exosuit repair droids and impact cushion plates on mecha with a machete to fix conventionally unkillable durands

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Slashes exosuit repair droids and impact cushion plates on mecha with a machete to fix conventionally unkillable durands.
The maximum armor you can get from three cushion plates is now 15% melee resistance.
The maximum amount of per tick healing you can get from three repair droids is now 0.5 healing a tick.

## Why It's Good For The Game

dreamseeker_TXvAarNjEq

This shit was absolutely decimating blobs along with generally being unhealthy for the balance of the game. Therefore, these changes are needed.

## Changelog
:cl:
balance: Slashes exosuit repair droids and impact cushion plates on mecha with a machete to fix conventionally unkillable durands.
balance: The maximum armor you can get from three cushion plates is now 15% melee resistance.
balance: The maximum amount of per tick healing you can get from three repair droids is now 0.5 healing a tick.
/:cl:

### CODE REVIEW

**Comment by Steel-Serperior** [comment]
Can we also remove mech RCD from combat mechs too?

**Comment by KingkumaArt** [comment]
As a mech lover, i'm fine with these changes, with the exception of the melee armor nerf. That durand dies to a laser gun even with armor as it stands now. Also yeah, the RCD has to go for durand and gygax.

**Comment by Iamgoofball** [comment]
maybe try mixing it up with some projectile armor too

**Comment by KingkumaArt** [comment]
> maybe try mixing it up with some projectile armor too

I'm curious to see how this performs with hits to kill, as it stands durands have such low laser armor you need to max it to have more than any other mech. I'm in support of this pr, but do wonder about the numbers.

I've said this many times before, I love mechs, and I'd hate if they were meta because then I couldnt love them.

**Comment by carlarctg** [comment]
i once spent 100 resources on a durand as blob and it got like 15% damage

merge

**Comment by SimplySmiley123** [comment]
> maybe try mixing it up with some projectile armor too

It doesn't do much of value considering you're a slow apartment block in a Durand.

**Comment by Dawnseer** [comment]
Why not just... make em unable to stack?

**Comment by KingkumaArt** [comment]
Just a suggestion before this convo gets heated - we could make the nerf less severe, and if its still a problem, increase it, rather than going full bore instantly. I feel a blob is kind of a bad metric to judge on, as blobs die to a single guy with an xray laser or an engineer wth an emitter.

**Comment by Arturlang** [comment]
Instead of nerfing the buffs that each component gives, nerf the amount a durand can get

**Comment by ElGitificador** [comment]
if the problem is with mechs and blobs a better solution would be to buff blobs damage against mechs instead of yet another nerf

**Comment by GPeckman** [comment]
> This shit was absolutely decimating blobs along with generally being unhealthy for the balance of the game. Therefore, these changes are needed.

So, just making sure I understand this PR correctly. You're nerfing mech armor values specifically to make blob more competitive against mechs?

**Comment by KingkumaArt** [comment]
> if the problem is with mechs and blobs a better solution would be to buff blobs damage against mechs instead of yet another nerf

Blobs are honestly in need of a full rework at this point. They live or die depending on if a quarter of the crew can be bothered to go kill them. Even with the reflective blob tiles, an emitter with a reflector still makes one an utter joke. Also depending if they sit in an asteroid on bird/wawa/tram and build up a gazillion resources and explode all at once you can just win, and its stupid.

**Comment by KingkumaArt** [comment]
> Also no shit a durand is gonna beat the tar out of a blob, its the only antag that cant move. The durand's biggest weakness - being slow as molasses - doesnt matter.

I'd also like to add to this the genuine best way to fight a durand is to just walk away from it and lure it into an open area. We mech users tend to have the power go to our heads and will likely follow you, at which point you can circle strafe us to death.

**Comment by zxaber** [comment]
> Why not just... make em unable to stack?

Honestly. Mech armor should be unstackable entirely, forcing to choose between melee or ranged armor.

**Comment by LSommers** [comment]
Wow, the third PR created due to the actions of a single person.
It's almost like administrative action should have been taken at this point.

**Comment by KingkumaArt** [comment]
> Wow, the third PR created due to the actions of a single person. It's almost like administrative action should have been taken at this point.

Gamers when they don't want admins to tell them to chill out with abusing features but also hate when those features get removed for everyone

**Comment by SimplySmiley123** [comment]
> > Also no shit a durand is gonna beat the tar out of a blob, its the only antag that cant move. The durand's biggest weakness - being slow as molasses - doesnt matter.
> 
> I'd also like to add to this the genuine best way to fight a durand is to just walk away from it, or try and lure it into an open area. We mech users tend to have the power go to our heads, and will likely follow you, at which point you can circle strafe us to death.
> 
> TLDR: If it sucks, hit da bricks.

You would be surprised how little thought spessmen can put into mech encounters.
Fighting an aparment complex sized block of metal on legs with guns for arms in a tiny maintenance shaft is definitely a better idea then doing literally anything else.

**Comment by DaydreamIQ** [comment]
Unfortunately this really won't stop durands from just taking t4 parts and tanking every hit imaginable with that energy shield of theirs. I swear something must've shadow buffed it at some point because it went from draining all your power in 5 seconds to lasting well over 5 minutes

**Comment by KingkumaArt** [comment]
> Unfortunately this really won't stop durands from just taking t4 parts and tanking every hit imaginable with that energy shield of theirs. I swear something must've shadow buffed it at some point because it went from draining all your power in 5 seconds to lasting well over 5 minutes

yes it was bugged to use 10x the power it was meant to, due to the megacell rework. it literally drained a cell from full to 0 with four hits from an esword.

**Comment by GuillaumePrata** [comment]
Durands should have low natural armor and just have the cool energy shield as their defense option, something that actually has the possibility of interesting couterplay around.

The idea of a walking Brickwall mech is fun in theory, but we will forever get stuck on new unfair nooks until we rework how our armor system itself works.

**Comment by CMDR-Gungnir** [comment]
Woah, you mean the tool designed to counter a specific antag type...countered that specific antag type? Shocking!

Mechs are good against Blobs, and to a lesser degree, Xenos, and that's...pretty much it. They're kinda shit against everything else, I've seen Man With Toolbox solo a mech before.

**Comment by LemonInTheDark** [comment]
> Blobs are honestly in need of a full rework at this point. They live or die depending on if a quarter of the crew can be bothered to go kill them. Even with the reflective blob tiles, an emitter with a box reflector still makes one an utter joke since it just shoots it right back.

This isn't a problem, it's kind of what we want to happen

**Comment by LemonInTheDark** [comment]
I don't really understand this, if we like the passive amount but not the stacking why not remove the stacking instead

**Comment by TheVekter** [comment]
As someone who's relatively mech-pilled, this is kind of an overtune based on one guy who's really good with them. The repair drones are kind of useless rn, this is just going to make them completely useless and bring back mechs dying to a stiff wind.

**Comment by CRITAWAKETS** [comment]
Probably would just be best to make the armor slot only one (armor stacking is dumb) or make it calculated multiplicatively. Making it only 5% is kind of ridiculous.

**Comment by CCC23** [comment]
I think the main problem here is the stacking armor not the armor itself specifically durands having 3 armor slots makes this a BIG problem but on lighter mechs like the paddy and gygax 1 melee armor or 2 melee armor on the gygax your not going to be defending when hits start flying your way

**Comment by Holinka4ever** [comment]
> Why not just... make em unable to stack?

See, it will not impact just people whom interact with that part of the game, so goof trying to pull them too, even if they are not the case of the problem.

Multiply module stacking is cancer indeed, there is a lot of way fixing it, from not allowing to stack, to debuffing module after second one.

But goof choose to literally piss off people whom making mechs, because situation right now is hot headed, and maintainers can merge any stuff that can "fix" fortnite mech.

**Comment by LemonInTheDark** [comment]
> Wow, the third PR created due to the actions of a single person. It's almost like administrative action should have been taken at this point.

two things can be true, someone can overuse behavior and said behavior can be a bit overtuned, if you want something to happen administratively please speak to the admin team instead of my pr comments.

**Comment by necromanceranne** [comment]
I'm going to be absolutely genuine when I say that this does not, in fact, solve the problem you think it solves.

I do think that repair droid stacking might be an issue, but I have an issue with the repair droid to begin with. It either makes mech healing as trivial as having power (as it does now since it can be stacked), or it makes no meaningful difference and mostly just exists to comfort less skilled pilots (as it did when it didn't stack and you only had the healing of a single drone, which is equivalent to the stacked drones here). Preferably, I think it shouldn't exist. I think being able to pressure mechs is one of their only meaningful weaknesses and I don't think passive healing helps that whatsoever.

I have no strong feelings about the armor plates. I just think it doesn't do anything to resolve the problem with blobs verses mechs. The reason I don't believe it helps against mechs is because I think mechs dying to a blob is entirely a failure on the part of the pilot, and that any pilot worth their salt simply doesn't let that happen. They don't need to, and mechs on their own can make up for the power of at least three crew members on their own. And it does not need to be a combat mech either. I've nearly solo'd blobs in Ripleys. Mechs are blob bane, because mechs are moving emitters. Blobs love to fuck up atmosphere, and mechs thrive in low atmosphere. The problem is with blobs being such good targets for such a wide variety of different tools.

It bears repeating every time this comes up; blob is badly designed, and has not had meaningful balance passes on itself in quite some time. You should not be balancing things around blob by nerfing a counter generally, affecting how it performs outside of blob as well. 

You should be balancing blob within blob itself, focusing on its own code and features. This will happen again, and it will look fairly similar to the problem here but in some other lick of paint. It is a Sisyphean task to try and nerf the rest of the game to make blob not die to the most middling of tools. And its also just lazy.

**Comment by MrMelbert** [comment]
Today I learned: When we gave mechas multiple armor slots a few years ago, we didn't change how much armor the modules gave, what were we thinking?

**Comment by MrMelbert** [comment]
I prefer that pr instead

**Comment by MrMelbert** [comment]
Actually to clarify I don't 100% prefer that PR

I think goof is right in that armor slots are kinda insane in their current implementation and think it could be addressed either way (slots are plentiful but provide smaller bonuses, or slots are few but provide larger bonuses)

I actually would like to see armor slots be used to specialized mechs even further, zxsaber suggested having the platings provide negative armor for the opposing type and I think that's cool (Like, you can pick from `+10 melee -5 bullet, +5 melee, +5 bullet, +10 bullet -5 melee` or something)

**Comment by SimplySmiley123** [comment]
> Durands should have low natural armor and just have the cool energy shield as their defense option, something that actually has the possibility of interesting couterplay around.
> 
> The idea of a walking Brickwall mech is fun in theory, but we will forever get stuck on new unfair nooks until we rework how our armor system itself works.

And then gets unused if it were to rely on an energy shield draining power to be worth a damn.
Considering the only place for you to get power from for a mech is robotics or deep within security it would be easy to just camp those, or pre-emptivly destroy and nullify the mech entirely.

**Comment by SimplySmiley123** [comment]
> Preferably, I think it shouldn't exist. I think being able to pressure mechs is one of their only meaningful weaknesses and I don't think passive healing helps that whatsoever.

You already can regardless of the mech's healing or armour solely because that mech has to get recharged eventually, with the only mech chargers on the station being easy to access and easy to deny to the mech with Ions and EMPs being piss easy to get and use on a mech that isn't a phazon too scared to fight a single greyshirt outside of maintenance. Technically plasma and uranium generator modules exist but they're not that great to use and don't benefit the pilot as much as negating the godawful malfunctions that just kill you regardless of the mech's integrity with those armour modules.
_Oh right yeah the malfunctions._ Gotta love the mech having a random chance of getting a non-issue malfunction or becoming unusable.

> and that any pilot worth their salt simply doesn't let that happen because they never need to approach the blob to kill it.

If a mech dies to a blob, it's either them getting quickly surrounded or the mech pilot is unlucky and rolled movement malfunction the second blob (or rather blobbernauts and spores) fought back and physically cannot try and get away.

> It bears repeating every time this comes up; blob is badly designed, and has not had meaningful balance passes on itself in quite some time. You should not be balancing things around blob by nerfing a counter generally, affecting how it performs outside of blob as well.

True. Blob is one humongous unmoving target, regardless of how often they move the core. Anything ranged will be a pain to a blob and anything exceptionally strong in melee range. Doesn't matter if it's a combat mech, a mining mech, a dude with a PKA, cargo spamming laser guns, someone with a god damn pipegun even can be a thorn in blob's side, everything that doesn't require the crewmember to directly melee fight the blob to do damage to said blob is gonna be good against the blob.

**Comment by GuillaumePrata** [comment]
> And then gets unused if it were to rely on an energy shield draining power to be worth a damn.

If only we had the power to rebalance drain values..
If it is draining too much, lower it, if it becomes too opressive, make them choose when to use the shield better instead of having it 24/7 by increasing the drain.

Or go in another angle, the shields can stay up for 2 minutes and consume no power, but them go on a 2 minute cooldown, now the Durand needs to go in and out of combat to stay efficient, which is a cooler design angle than a brick wall walking forward endlessly.

**Comment by Timberpoes** [comment]
Mech combat uptime was one of the factors around the generally controversial changes to the old Xenobio yellow core power cells I did over 4 years ago in #54198.

4 years past and I think I can put into words the kind of niche I think I like mechs filling the most and it;s very similar to the mech justifications in that old PR.

I like the idea that the niche of combat mechs is short term power projection. That you bring them out if you need a quick burst of firepower. That means combat mechs shouldn't exist as a default state of someone's being. It shouldn't be viable to use them as if they were a mere exoskeleton, suited for 24/7 use across the station.

I don't even think it's unfair for them to have zero "skillful" counterplay if they can be counterplayed by avoiding their strengths (short-term power projection) while exploiting their weaknesses (long-term frontline sustainability).

So how I'd approach this would be to totally transition away from the idea of stacking armour and repairs at all and move to mech defenses being primarily energy shielded. This allows mechs to be at their strongest on first deployment and at their weakest as combat progresses. Their cell charge should be their currency. This indirectly makes EMPs a more enticing option against mechs as draining their cell charge is draining their time on the battlefield in a meaningful way, if not outright leading to the mech's destruction.

That downtime design allows mechs to fill that awesome power fantasy in a way that is sustainable with the rest of the game. It means targetting the power infrastructure can weaken a mech's long-term viability while not impacting a fully charged and kitted mech's power fantasies.

Contrasted with the fleshbag crew, whom can sustain longer sieges through more efficient field medicine, more flexible and more energy efficient weaponry, etc - it introduces a good asymmetry that makes for such good David vs Goliath moments.

This means that a mech can obliterate a threat, but it cannot obliterate all threats without it needing to be withdrawn from the frontlines for repair, recharge and general maintenance. Or be abandoned by its pilot once it has served its purpose.

That's my take anyway.

**Comment by GuillaumePrata** [comment]
> That's my take anyway.

And the best take we can ever have in mech designs inside our sandbox IMO.

If we were to use any basis for good mech vibes, Evangelion already has us covered, the moment the power cord is removed the mechs have a hard limit of time before they become a powerless pieces of "metal" there and it is fitting.

Mechs should be giant power fantasy multipliers were the pilots drop everything they are doing and run into during emergencies, not something you hop in and out all shift just waiting for a valid to show up.

**Comment by zxaber** [comment]
It was a lot easier back when you could leave a mech parked and be relatively sure it would stay where you parked it.

Now that removing DNA/ID lock is as easy as hacking a door it's hard to go through the effort of making a mech and then letting it sit out of sight.

**Comment by GuillaumePrata** [comment]
> Now that removing DNA/ID lock is as easy as hacking a door it's hard to go through the effort of making a mech and then letting it sit out of sight.

That is true, but I also think we always had a weird system for how we lock mechs, before it was nigh impossible to steal a mech, now it is too trivial, we need a middle ground.

_Clearly_ what we need is mech car keys, now you can easily steal someone's combat mech if you kill them out of it, and there is always the chance someone forgets to take their keys out and someone just hop in and walk away, borrowing for someone you trust, etc, etc.

**Comment by SimplySmiley123** [comment]
> So how I'd approach this would be to totally transition away from the idea of stacking armour and repairs at all and move to mech defenses being primarily energy shielded. This allows mechs to be at their strongest on first deployment and at their weakest as combat progresses. Their cell charge should be their currency

That sounds good on paper but with how trivial it is to drain power off of devices, weapons and mechs either by security outright starting with a gun perfect for this kinda job, quite easy to use, relatively safe if not used at point blank range, or someone creating an EMP grenade (or not even bothering to do that and just mixing iron and uranium near whatever they want EMP'd) it would be nigh pointless to try without a lot more changes to stuff that isn't just the mechs.

Plus the problem of mech chargers and how far out you can get away from them with this sort of idea.
Durand especially wouldn't be able to get far out from science, plus considering how trivial it is to steal a mech now, making more mech charger areas around the station is an extreme pain to make it even remotely secure while still reasonably accessible for the mech.
Would require better ways to secure a mech that also don't just make it completely safe to leave a mech out in the open without any superstitions around it.

Pluuuuus it's too cumbersome for emergencies with how they are right now. it takes a good while to build them, takes a while to get into one and do prep before going out (eeespecially with Savannah Ivanov since it's not just you who needs to be ready, but also another person, as well as Durand with how slow it is ironically enough considering these nerfs are because of the Durand), then by the time you reach where you wanted to go the situation's already been handled or got too out of control for the mech to matter, mostly why I personally stay in the mech for prolonged periods of time walking around the station (of course gotta get out at some point to deal with hunger, to resupply on ammo, welding fuel, etc.) because unless you're already prepped and ready to go you ain't gonna respond in time and get in on the action. Unless it's blob, dragon bothering with the rifts or a cultist megabase kind of situation where the threat isn't gonna run around all over the station.

> Or go in another angle, the shields can stay up for 2 minutes and consume no power, but them go on a 2 minute cooldown, now the Durand needs to go in and out of combat to stay efficient, which is a cooler design angle than a brick wall walking forward endlessly.

Something like this would be a lot less frustrating and a single assistant that had access to the ORM for a second wouldn't immediately shut it down. You can already lock a mech down in place how it is currently with the ion gun or random John Grey mixing contents of two beakers together in your general area. It's trivial to shut down something centered around requiring power to work.
Plus this wouldn't be a blanket change affecting all the other 3 combat mechs, cuz Durand ain't the only mech that exists.
